web/controllers/certificateController/index.ts

```diff
@@ -439,9 +439,6 @@ function loadTemplate(name, lang: Language, fallback: boolean = true): EJS.Clien
 
     let path = `./assets/${name}.${lang}.html`;
 
-    if (process.env.NODE_ENV == 'dev')
-        path += `.example`;
-
```

Comment:
If we remove these lines, why do we still need the sample files? If we remove them, users running the backend on their machines will see an exception. I think we should rename these files then and remove the .example at the end. Or just leave this line in and somehow add `&& IS_HEROKU !== TRUE`.

And by the way, the Drehtür certificates also use this `.example` stuff, which then should be changed too, whatever solution we choose, see https://github.com/corona-school/backend/blob/fc7d7b701430d0d1dbb84d71119948c7cfe0163c/common/drehtuer/certificates.ts#L18-L20

Reply:
true, that was laziness ...